modules/osdk-updating-v125-to-v128.adoc

```diff
@@ -41,10 +41,10 @@ The following procedure updates an existing {type}-based Operator project for co
 .Procedure
 
 ifdef::helm,hybrid,java[]
-* Find the `ose-kube-rbac-proxy` pull spec in the following files, and update the image tag to `v4.13`:
```

Comment:
@adellape @michaelryanpeter Can you PTAL at the changes in this file, since I made more changes so that the bump actually makes sense

Reply:
Thank you!!